core/src/test/java/io/questdb/cutlass/pgwire/PGJobContextTest.java

```diff
@@ -2995,6 +2995,35 @@ public void testInvalidateWriterBetweenInserts() throws Exception {
         });
     }
 
+    @Test
+    public void testUpdateAfterDropAndRecreate() throws Exception {
```

Comment:
this is insufficient; there should be a test, which removes column `update` depends on. The error message should be adequate

Reply:
The tests are added.